packages/main/src/plugin/openshell-cli/openshell-gateway.ts

```
@@ -211,13 +220,18 @@ export class OpenshellGateway implements Disposable {
+  async checkAvailable(): Promise<void> {
+    return this.#availablePromise.promise;
+  }
+
   isRunning(): boolean {
     return this.#gatewayProcess !== undefined && typeof this.#gatewayProcess.exitCode !== 'number';
   }
 
   @preDestroy()
   dispose(): void {
     this.stop().catch((err: unknown) => console.error('[openshell-gateway] failed to stop: ', err));
+    this.#disposables.forEach(disposable => disposable.dispose());
```

Biome (2.5.1)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)